### PR TITLE
Add name Jorge Araya Torres

### PR DESCRIPTION
**Pull request by Jorat2211**

### CODE REVIEW

**Comment by crojasaragonez** [comment]
please fix the conflicts

**Comment by Jorat2211** [comment]
> please fix the conflicts

Complete fix the conflicts

File: names.txt [hunk 1]
```diff
@@ -5,6 +5,10 @@ Oscar Antonio Arias Vargas
 Daniel Vargas Sibaja
 Esteban Murillo Rojas
 Luis Carlos Segura Barboza
+
```
Comment (crojasaragonez) [inline names.txt]: remove these empty lines

**Comment by crojasaragonez** [comment]
fix the conflicts please

**Pull request merged**